dapps/appkit-siwe/react/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "appkit-siwe-react",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@tanstack/react-query": "^5.51.1",
+    "@wagmi/core": "^2.11.7",
+    "@web3modal/siwe": "^5.0.6",
+    "@web3modal/wagmi": "^5.0.4",
```

```suggestion
    "@web3modal/wagmi": "5.0.6",
```

All versions should be fixed and in latest